client/src/main/java/io/narayana/lra/client/internal/NarayanaLRAClient.java

```diff
@@ -333,12 +331,6 @@ public URI startLRA(URI parentLRA, String clientID, Long timeout, ChronoUnit uni
             Current.addActiveLRACache(lra);
 
             return lra;
-        } catch (UnsupportedEncodingException uee) {
```

Comment:
I am not sure why we are no longer catching UnsupportedEncodingException?

Reply:
Resteasy converts it to a BadRequestException (HTTP code 400).

BTW I forget the actual reason why I removed it so I think it is safer to reinstate the catch block.

Reply:
OK, thank you

Reply:
@tomjenkinson  Ah wait, the exception is reported by calling name() on the charset (StandardCharsets.UTF_8.name()) but since I removed the call to name() the catch isn't valid anymore - see my response to your earlier question about this.